coderd/database/migrations/000199_ensure_default_everyone_group.up.sql

```diff
@@ -0,0 +1,11 @@
+-- This ensures a default everyone group exists for default org.
+INSERT INTO
+	groups(name, id, organization_id)
+SELECT
+	-- This is a special keyword that must be exactly this.
+	'Everyone',
+	-- Org ID and group ID must match.
+	(SELECT id FROM organizations WHERE is_default = true LIMIT 1),
+	(SELECT id FROM organizations WHERE is_default = true LIMIT 1)
```

Comment:
Is `LIMIT 1` required for SQL to correctly cast a rowset into a scaler?  The unique index on `is_default` should ensure there is exactly one organization where it is `true`.

Reply:
It is not required, the unique index should protect this.

I added the `LIMIT 1` for defensive programming in our migrations since they can prevent boot of Coderd.